arq/worker.py

```diff
@@ -247,6 +251,9 @@ def __init__(
         self.job_serializer = job_serializer
         self.job_deserializer = job_deserializer
 
+        if timezone is not None and isinstance(timezone, str):
+            os.environ[arq_timezone] = timezone
```

Comment:
why are you doing this? This is not a good way to shared variables globally.

Either use an actual global variable, or use an actual environment variable.

This is an antipattern.